.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Validate & Release
+on:
+  push:
+    tags:
+      - v*
```

Comment:
Should we limit this to a specific branch? main or maybe a new release branch?
```yml
push:
    branches:
      - main
    tags:
      - v*
```

Should this also look for a pull request action?

Reply:
Just on push tags is fine. Releases aren’t always from the main branch. Sometimes you want to release alpha versions for example. These could be don’t off of a release-candidate or “next” branch. I think for now the way you have it is great. We also wouldn’t want to run on pull requests automatically. What if the PR was broken and released a breaking version!